MultiplayerMod/Core/Patch/HarmonyArgsSupport.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using MultiplayerMod.Core.Extensions;
+using MultiplayerMod.Core.Logging;
+
+namespace MultiplayerMod.Core.Patch;
+
+// ReSharper disable once UnusedType.Global
+[HarmonyPatch(typeof(MethodPatcher))]
+[HarmonyPriority(Priority.First)]
+public static class HarmonyArgsSupport {
```

Comment:
Could you add class comments that this is a workaround since ONI using outdated Harmony? Thanks!